src/RestWrite.js

```diff
@@ -1121,7 +1121,12 @@ RestWrite.prototype.runAfterTrigger = function() {
   this.config.liveQueryController.onAfterSave(updatedObject.className, updatedObject, originalObject);
 
   // Run afterSave trigger
-  return triggers.maybeRunTrigger(triggers.Types.afterSave, this.auth, updatedObject, originalObject, this.config);
+  return triggers.maybeRunTrigger(triggers.Types.afterSave, this.auth, updatedObject, originalObject, this.config)
+    .catch(function(err) {
+      /* eslint-disable no-console */
+      console.warn('afterSave caught an error: ' + err);
```

Comment:
we should probably use the logger so we have structured logging etc.

Reply:
```
logger.warn('afterSave caught an error', err);
```
so if there's an error, maybeRunTrigger will now resolve.  Is that the right behavior?  Should we return a rejected promise?  Not a request, a genuine wrapping my head around promises question.....

Reply:
I'll put the logger call in, I was being a bit lazy perhaps... As for the issue of handling the promises the docs indicate that, in no case, should the failure be returned to the user. All errors should be in the logs. Unless we want to change the doc of course.

I'm open to consider any alternatives as well to handling failure conditions here. There's probably a few other approaches besides catching whatever happens to come back.